docs/okeps/okep-5088-evpn.md

```diff
@@ -0,0 +1,789 @@
+assigning an VTEP IP to each node. If the cidr range includes the Kubernetes node IP for a node, then the node IP will be
+used for VTEP IP. Note, using the node IP should be avoided in most cases, as that IP will already be tied to a specific
+interface on the node. This prevents proper Layer 3 failure handling. While the node IP on a dedicated link could use
+bonding to prevent Layer 2 failover, if something goes wrong on the Layer 3 interface, or the leaf connected to that
+link goes down, then there is no failover.
+With EVPN, it is advantageous to assign the VTEP IP to a loopback interface, so that multihoming
+and failover handling can occur. If a link goes down to one leaf, BFD will fire, and there will be a mass withdrawal of
+routes, moving all traffic to a second leaf.
+
+ovnkube-cluster-manager will handle annotating the node with the assigned IP address. If the VTEP is in unmanaged mode,
+then ovnkube-cluster-manager will only handle checking that all nodes have an IP address annotated for this VTEP. If a
+node is missing the IP, the VTEP will be updated with a failure status condition.
+
+For unmanaged, the ovnkube-node component will handle detecting the IP address on the Linux node and setting the node
+annotation.
+
+#### EVPN
+
+When a UDN is created with EVPN configuration, the UDN controller in ovnkube-cluster-manager will check to ensure that
+a VTEP exists for this EVPN. If one does not, then the NAD will not be rendered and the UDN will be put into error state.
+Additionally, a check will be done to ensure that VNIs do not overlap between any other UDNs in the cluster.
+
+Once the NAD has been rendered, ovnkube-controller and ovnkube-node network controllers will be started. The ovnkube-node
+network controller will detect that this network is EVPN enabled, and then create the correct network device configuration
+in Linux.
+
+The BGP RA controller will be responsible for detecting when a BGP RA selects a CUDN that is EVPN enabled. Once it does,
+it will generate the proper FRR-K8S configuration.
+
+For the rest of the examples in this section, assume there is a layer 2 UDN called "blue", with subnet 10.0.10.0/24.
+
+##### Node Configuration: MAC-VRF + IP-VRF Combination with Layer 2 UDN
+
+Once the VTEP IP is assigned, ovnkube-node will then handle configuring the following:
+```bash
+# VTEP IP assignment to loopback - only done in VTEP managed mode
+ip addr add 100.64.0.1/32 dev lo
+
+# SVD bridge + VXLAN setup
+ip link add br0 type bridge vlan_filtering 1 vlan_default_pvid 0
+ip link set br0 addrgenmode none
+ip link set br0 address aa:bb:cc:00:00:64
+ip link add vxlan0 type vxlan dstport 4789 local 100.64.0.1 nolearning external vnifilter
+ip link set vxlan0 addrgenmode none master br0
+ip link set vxlan0 address aa:bb:cc:00:00:64
+ip link set br0 up
+ip link set vxlan0 up
+bridge link set dev vxlan0 vlan_tunnel on neigh_suppress on learning off
+
+# Create the IP-VRF
+# Map VLAN 11 <-> VNI 101
```

Comment:
Am I correct that the VLAN # (in this case 11) is "local" to this network VTEP? IOW, The VLAN is only used for separation inside the VTEP, and between VTEPs it is the VNI that does it.